source/site-administrators/studio/bucket-storage-publishing.rst

```diff
@@ -0,0 +1,178 @@
+:is-up-to-date: True
+
+.. index:: Bucket Storage Publishing
+
+.. _bucket-storage-publishing:
+
+=========================
+Bucket Storage Publishing
```

Comment:
@sumerjabri do we want to use the word `bucket` to describe the feature? I think that makes it sound S3 specific. 

@alhambrav maybe we can add a list of supported stores, for example:
```
Supported blob stores:
- AWS S3 (maybe a link for specific configuration)
```

That way we can easily add new implementations in the future without having to change this docs

Reply:
Let's get away from the word bucket if possible.